<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wizard</class>
 <widget class="QWizard" name="Wizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="QWizardPage" name="IntroPage">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Introduction</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>It seems that the configuration does not work or is incorrect.
Fill in the fields on the next wizard page and try the connection to the database.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="DbConfigPage">
   <property name="title">
    <string>Database configuration</string>
   </property>
   <property name="subTitle">
    <string>Specify information on how to connect to db</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="11" column="0">
     <widget class="QLabel" name="databaseStatusLabel">
      <property name="text">
       <string>Database Status:</string>
      </property>
      <property name="buddy">
       <cstring>databaseStatusResult</cstring>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QPushButton" name="testConnection">
      <property name="text">
       <string>&amp;Test</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="usernameLabel">
      <property name="text">
       <string>User name:</string>
      </property>
      <property name="buddy">
       <cstring>usernameLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="databaseLabel">
      <property name="text">
       <string>Database name:</string>
      </property>
      <property name="buddy">
       <cstring>databaseLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="driverLabel">
      <property name="text">
       <string>Database driver:</string>
      </property>
      <property name="buddy">
       <cstring>driverComboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="connectionStatusLabel">
      <property name="text">
       <string>Connection status:</string>
      </property>
      <property name="buddy">
       <cstring>connectionStatusResult</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="hostnameLabel">
      <property name="text">
       <string>Hostname:</string>
      </property>
      <property name="buddy">
       <cstring>hostnameLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="11" column="3">
     <widget class="QPushButton" name="createDB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&amp;Create DB</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QCheckBox" name="showPasswordCheck">
      <property name="text">
       <string>&amp;Show password</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Password:</string>
      </property>
      <property name="buddy">
       <cstring>passwordLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>password</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QLineEdit" name="usernameLineEdit">
      <property name="placeholderText">
       <string>user</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLineEdit" name="databaseLineEdit">
      <property name="placeholderText">
       <string>my_database</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="hostnameLineEdit">
      <property name="placeholderText">
       <string>www.example.com</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QComboBox" name="driverComboBox"/>
    </item>
    <item row="10" column="1" colspan="2">
     <widget class="QLineEdit" name="connectionStatusResult"/>
    </item>
    <item row="11" column="1" colspan="2">
     <widget class="QLineEdit" name="databaseStatusResult"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="conclusionPage"/>
 </widget>
 <tabstops>
  <tabstop>driverComboBox</tabstop>
  <tabstop>hostnameLineEdit</tabstop>
  <tabstop>databaseLineEdit</tabstop>
  <tabstop>usernameLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>connectionStatusResult</tabstop>
  <tabstop>databaseStatusResult</tabstop>
  <tabstop>showPasswordCheck</tabstop>
  <tabstop>testConnection</tabstop>
  <tabstop>createDB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
